docs/cookbook/using-docker.md

```diff
@@ -280,8 +280,21 @@ APOS_S3_SECRET=<account secret>
 
 </AposCodeBlock>
 
-While setting S3 permissions is beyond the scope of this guide, you might have to either change your `@apostrophecms/uploadfs` options for accessing the bucket or make changes to the ACL policies through your AWS dashboard.
 ### Finishing up
+While our Docker container is now configured for storing items on AWS S3, it won't fully work if we were to spin it up now. First, we have to configure our S3 bucket to allow our site to access it. This is easily done through the AWS control panel.
```

Comment:
allow our site to access it -> allow the public to access it

Reply:
Yeah, dumb mistake. Corrected.